README.md

```diff
@@ -1,71 +1,109 @@
+
+<div align = center >
+
 # Watlings
 
-Learn WebAssembly Text Format by fixing a bunch of small programs!
+Learn the WebAssembly Text Format  
+by fixing a bunch of small programs!
+
+<br>
+
+[![Button Roadmap]][Roadmap]
+
+<br>
 
-![watlings file 001_hello](https://github.com/EmNudge/watlings/assets/24513691/a720199d-75ce-41c5-84d6-014ef2b2eacd)
-![console output for failing test](https://github.com/EmNudge/watlings/assets/24513691/1aa1e029-f871-41d6-a5fd-44997d82b148)
+<img
+    width = 400
+    src = https://github.com/EmNudge/watlings/assets/24513691/a720199d-75ce-41c5-84d6-014ef2b2eacd
+/>
 
+<img
+    width = 400
+    src = https://github.com/EmNudge/watlings/assets/24513691/1aa1e029-f871-41d6-a5fd-44997d82b148
+/>
+
+</div>
+
+<br>
+<br>
 
 ## 🚧 Caution
```

Comment:
move to [alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
```suggestion
> [!WARNING]
> This project is incomplete and in active development.  
> Feel free to help out by filing issues and creating PRs!
```